### DISCUSSION TITLE
Integrating OpenAI assistant, where is order receiving implementation?

### DISCUSSION THREAD

**Discussion by Max77788** [opensourcepos/opensourcepos#3890] [Q&A]
Hi, I would love to integrate the AI assistant into the OSPOS, but I struggle for now with finding the file where order registering is implemented. E.g. def accept_order("item", "quantity"). Where may I find this chunk of code? Appreciate the navigation and aid)

**Comment by louis030195** [comment]
https://github.com/stellar-amenities/assistants @Max77788
